.github/workflows/pr-screenshots.yml

```
@@ -0,0 +1,103 @@
+          else
+            echo "No migrations found, initializing database..."
+            uv run python ../scripts/init_db.py
+          fi
+      
+      - name: Start services
+        run: |
+          # Start backend
+          cd backend
+          uv run uvicorn app.main:app --host 0.0.0.0 --port 8000 &
+          cd ..
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)